test_cases/example_2.json

```diff
@@ -1,1164 +1,925 @@
-              "SequenceOrder": 1
-            },
-            {
-              "CandidateIds": [
-                "candidate_gomez"
-              ],
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_gomez",
-              "SequenceOrder": 2
-            },
-            {
-              "CandidateIds": [
-                "candidate_davis"
-              ],
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_davis",
-              "SequenceOrder": 3
-            },
-            {
-              "CandidateIds": [
-                "candidate_tsi"
-              ],
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_tsi",
-              "SequenceOrder": 4
             },
-            {
-              "CandidateIds": [
-                "candidate_orotund"
-              ],
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_orotund",
-              "SequenceOrder": 5
-            },
-            {
-              "IsWriteIn": true,
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_4.6",
-              "SequenceOrder": 6
-            },
-            {
-              "IsWriteIn": true,
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_4.7",
-              "SequenceOrder": 7
-            },
-            {
-              "IsWriteIn": true,
-              "@type": "ElectionResults.CandidateSelection",
-              "@id": "selection_4.8",
-              "SequenceOrder": 8
-            }
-          ],
-          "ElectionDistrictId": "district_unified_school_district",
-          "Name": "Gadget County School Board",
-          "VoteVariation": "n-of-m"
+            "@type": "ElectionResults.Office"
         },
         {
-          "@id": "contest_6",
-          "ContestSelection": [
-            {
-              "Selection": {
+            "@id": "vpotus",
+            "Name": {
                 "Text": [
-                  {
-                    "Content": "yes",
-                    "Language": "en",
-                    "@type": "ElectionResults.LanguageString"
-                  }
+                    {
+                        "Content": "Vice President of the United States",
+                        "Language": "en",
+                        "@type": "ElectionResults.LanguageString"
+                    }
                 ],
                 "@type": "ElectionResults.InternationalizedText"
-              },
-              "@type": "ElectionResults.BallotMeasureSelection",
-              "@id": "selection_6.1",
-              "SequenceOrder": 1
             },
-            {
-              "Selection": {
+            "@type": "ElectionResults.Office"
+        }
+    ],
+    "Party": [
+        {
+            "@id": "lepton_party",
+            "Name": {
```

Comment:
My parsing logic makes an assumption that all parties will have an `Abbreviation`. It therefore fails because the parties in this EDF only have a `Name`.

Based on the [spec](https://pages.nist.gov/ElectionResultsReporting/#17_0_2_4_78e0236_1389366278128_412819_2460), this model is correct. That tells me that my logic needs to change, because `Abbreviation` can be nullable - whereas `Name` must always exist.

What should the parsing logic look like here - what should we display to the voter? cc @trustthevote

Reply:
The spec says, of the required `Name` attribute:

>Official full name of the party, e.g., “Republican”; can appear on the ballot.

If the official name of the party is "The Lepton Party of Farallon", it seems to me that the EDF is obligated to make that the value of the Name field, regardless of whether or not it fits on the form.  HOWEVER, ElectionResults.InternationalizedText has an optional `Label` attribute, and I think that's what you should be using.

In fact, I'd say that we should be providing a `Label` for every displayable Text, and that you should use the Label whenever it is present.

I have made it so in the file.